### PR TITLE
week 2 takeaway challenge

### PR DESCRIPTION
**Pull request by sabrinalord**

### CODE REVIEW

File: spec/menu_spec.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+subject(:menu) { Menu.new(menu_items) }
+    let(:menu_items) do
+    {
+        pizza: 8, 
+        ramen: 6, 
+    }
+    end
+
+    it 'stores a list of dishes and their prices' do
+    expect(menu.menu_items).to eq(menu_items)
```
Comment (j-cort) [inline spec/menu_spec.rb:13]: clear and accurate unit tests!

File: spec/order_spec.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+require 'order'
+require 'takeaway'
+
+describe Order do
+    
+    # it 'stores my selected dish' do
```
Comment (j-cort) [inline spec/order_spec.rb:6]: Seems like a good test. Why comment it out?

File: lib/takeaway.rb [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+      @menu.print
+    end
+
+    def add_to_order(dish, quantity)
+      @order.my_dishes << {dish => quantity}
+      puts "#{quantity} #{dish} added to your order!"
+    end 
+
+end 
+
```
Comment (j-cort) [inline lib/takeaway.rb:22]: I like that you used many different classes/module to separate the responsibilities of the program. I think this will make it easier to expand the functionality of the program in the future.

File: spec/takeaway_spec.rb [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  describe '#add_to_order' do
+    let(:order) { double(Order.new)}
+    let(:dish) { "pizza" }
+    let(:quantity) {3}
+        it 'adds the dish and its quantity into my order' do 
+        subject.add_to_order(dish, quantity)
+        expect(subject.order.my_dishes).to include({dish => quantity })
+        end
+  end  
+end
```
Comment (j-cort) [inline spec/takeaway_spec.rb:25]: Great tests! How about also testing for any edge cases, e.g. if an item isn't on the menu., but a user tries to add it.
